### ISSUE TITLE
arabic font support 

### ISSUE THREAD

**Issue by abdo-spices** [excalidraw/excalidraw#6619]
When I write anything in Arabic, the font is bad. I want support for the Arabic font. You can use this font from [Google Fonts](https://fonts.google.com/specimen/Baloo+Bhaijaan+2?subset=arabic&noto.script=Arab)

**Comment by Mohsen-94** [comment]
Yes, the Arabic font renders correctly, but it does not look good. It looks too standard and basic, which doesn't go well with the hand-drawn aesthetic.

**Comment by hulkbig** [comment]
@blackPeppper hey, you can try my fork version: [https://excalidraw-zh.com/](https://excalidraw-zh.com/)

**Comment by blackPeppper** [comment]
@hulkbig why

**Comment by hulkbig** [comment]
@blackPeppper my fork version support you to custom your favorite fonts, detail in: [https://github.com/hulkbig/excalidraw-zh](https://github.com/hulkbig/excalidraw-zh)

**Comment by DawidWraga** [comment]
Incase anyone is wondering, I spent a while figuring this out but I've found a solution.

I've written an article on how to use custom fonts on excalidraw [here](https://dev.to/dawidcodes/how-to-use-a-custom-font-on-excalidrawcom-4jl4)
